mitmproxy/tools/main.py

```diff
@@ -34,134 +35,80 @@ def assert_utf8_env():
         sys.exit(1)
 
 
-def process_options(parser, options, args):
+def process_options(parser, opts, args):
     if args.version:
         print(debug.dump_system_info())
         sys.exit(0)
+    if args.options:
+        print(optmanager.dump(opts))
+        sys.exit(0)
+    if args.quiet:
```

Comment:
The handling of quiet here is unchanged from before the patch. I _think_ the idea here is that -q should take precedence over any -v flags, regardless of the order they were passed in. 

I think we need to clarify this anyway. I'd like verbosity to be a string that matches our log level names, which is clearer.